endpoint/endpoint_diagnoseComp/endpoint_diagnose.h

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ENDPOINT_DIAGNOSE_H
+#define ENDPOINT_DIAGNOSE_H
+
```

Comment:
No content?

Reply:
Now it has the content which is a single function for hardware logging.